src/util/language.js

```diff
@@ -0,0 +1,15 @@
+const franc = require('franc-all');
+const iso6393 = require('iso-639-3');
```

Comment:
We can make this more efficient by using the map from `require('iso-639-3/to-1')` to find the equivalent ISO 639-1 code from the ISO 639-3.